lib/travis/model/request/states.rb

```diff
@@ -53,6 +53,9 @@ def finish
     end
 
     def add_build
+      if config && config[:language]
+        config[:language] = config[:language].to_s.downcase
```

Comment:
Isn't `config[:language]` supposed to be a symbol, rather than a string? Or is it normalized somewhere downstream from here?

Has this been tested?